test/services/internalEngine/bootstrap.test.js

```diff
@@ -397,7 +397,7 @@ describe('services/internalEngine/bootstrap.js', () => {
   describe('#_waitTillUnlocked', () => {
     it('should throw if locked for too long', () => {
       bootstrap.db.exists.returns(Bluebird.resolve(true));
-      mockrequire('bluebird', Object.assign(require('bluebird'), {delay: () => Bluebird.resolve()}));
+      mockrequire('bluebird', Object.assign({}, require('bluebird'), {delay: () => Bluebird.resolve()}));
```

Comment:
That one bug took me so much time to find: it prevented unit tests on the new mutex class to work properly, because Bluebird.delay was overwritten by this Object.assign command, impacting the require cache for all following tests. :expressionless: